neofetch

```diff
@@ -1032,6 +1032,11 @@ get_distro() {
                     *) distro="Guix System $(guix system -V | awk 'NR==1{printf $5}')"
                 esac
 
+            # Display whether using '-current' or '-release' on OpenBSD.
+            elif [[ $kernel_name = OpenBSD ]] ; then
+                read -ra distro <<< "$(sysctl -n kern.version)"
+                distro=${distro[@]:0:2}
```

Comment:
Store the array with a different name to fix shellcheck. :+1:

Reply:
Oops.... I used the same name on purpose to "avoid creating an unneeded variable" :rofl: 

I'll change it

Reply:
```
SC2124: Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.
```

Try with `[*]` or even `${var[0]} ${var[1]}`